frontend/components/create-learning-circle-form.jsx

```diff
@@ -86,7 +101,7 @@ export default class CreateLearningCircleForm extends React.Component {
         config: { headers: {'Content-Type': 'application/json' }}
       }).then(res => {
         if (res.data.status === 'created') {
-          window.location.href = LC_PUBLISHED_PAGE;
+          window.location.href = `${LC_PUBLISHED_PAGE}?url=${res.data.study_group_url}`;
```

Comment:
Currently success pages redirects to the facilitator dashboard page rather than the direct learning circle page. Since most facilitators only have 1 active learning circle, this simplifies navigation